packages/manager/modules/pci/src/projects/project/instances/start/start.module.js

```diff
@@ -0,0 +1,24 @@
+import angular from 'angular';
+import '@ovh-ux/ng-translate-async-loader';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import 'ovh-api-services';
+
+import startInstance from '../instance/start/start.module';
```

Comment:
I see here, that you re-use the `start` module from `instances/instance`, great ! 

I don't remember if we do that elsewhere or if we do something like declare a component in a module and use it at 2 differents places (routing defined only). 🤔

Reply:
Ok we do that in a lot of module. That's ok.